float32[] ee_pose
---
bool done